tests/admin_views/tests.py

```diff
@@ -3743,11 +3786,12 @@ def test_add_model_modeladmin_defer_qs(self):
                                     post_data, follow=True)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(CoverLetter.objects.count(), 1)
-        # Message should contain non-ugly model verbose name
+        # Message should contain non-ugly model verbose name and link to edit form
```

Comment:
Please add period at end of sentence. Comments should be wrapped at 79 characters too.